autoload/go/debug.vim

```diff
@@ -899,6 +911,49 @@ function! s:isActive()
   return len(s:state['message']) > 0
 endfunction
 
+" List All Goroutines Running
+function! go#debug#Goroutines(...) abort
+  try
+    let l:currentGoroutineId = 0
+    try
+      let l:currentGoroutineId = s:groutineID()
+    catch 
+      echom "current goroutineId not found..."
+    endtry
+
+    let l:res = s:call_jsonrpc('RPCServer.ListGoroutines')
+    let l:goroutines = l:res.result.Goroutines
+    if len(l:goroutines) == 0
+      echom "No Goroutines Running Now..."
+      return
+    endif
+
+    for l:idx in range(len(l:goroutines))
+      let l:goroutine = l:goroutines[l:idx]
+      if l:goroutine.id == l:currentGoroutineId
+        echom "* Goroutine " . l:goroutine.id . " - " . l:goroutine.userCurrentLoc.file . ":" . l:goroutine.userCurrentLoc.line .  " " l:goroutine.userCurrentLoc.function.name . " " . " (thread: " . l:goroutine.threadID . ")"
```

Comment:
Can you change `echom` here to `go#util#EchoInfo`?

Reply:
fixed